jina/resources/helloworld.html

```diff
@@ -23,6 +23,7 @@ <h2 style="color: #1E6E73">What just happened?</h2>
         </div>
         
         <hr style="border: 1px dashed #009999;">
+        <th align="left"><h3 style="color: #1E6E73">Label Evaluation {% EVALUATION %}</h3></th>
```

Comment:
We can add here x% matched true label in top 5 result

Reply:
I prefer to be like this in case we ever change eval_at and do not remember

Reply:
I think the % number itself doesn't give clarity on the evaluation done and needs some definition. For auto-update, you can load the yaml and pass its eval_at to this string